### DISCUSSION TITLE
How to install Ticker updates?

### DISCUSSION THREAD

**Discussion by aqsmith08** [achannarasappa/ticker#72] [Q&A] [answered]
Hey All,

I installed Ticker a couple days ago onto my Linux laptop following the instructions in the README file:
```
curl -Ls https://api.github.com/repos/achannarasappa/ticker/releases/latest \
| grep -wo "https.*linux-amd64*.tar.gz" \
| wget -qi - \
&& tar -xf ticker*.tar.gz \
&& chmod +x ./ticker \
&& sudo mv ticker /usr/local/bin/
```

I'd like to update Ticker to the latest version but I'm unclear how to do that exactly. (Yep, I'm a noob with command line and know the problem is likely me.) I tried running the same command again but receive this output:
```
 curl -Ls https://api.github.com/repos/achannarasappa/ticker/releases/latest \
> | grep -wo "https.*linux-amd64*.tar.gz" \
> | wget -qi - \
> && tar -xf ticker*.tar.gz \
> && chmod +x ./ticker \
> && sudo mv ticker /usr/local/bin/
tar: ticker-2.2.0-linux-amd64.tar.gz: Not found in archive
tar: ticker-2.3.0-linux-amd64.tar.gz: Not found in archive
tar: Exiting with failure status due to previous errors
```

Any ideas how I can reinstall or update Ticker to get the latest functionality? Thanks!

**Comment by achannarasappa** [comment, marked as answer]
Looks like you have the old and new version of the tar in the same directory. Try removing all existing archives (`rm ticker*.tar.gz`) and running the linux install command again

**Reply by aqsmith08** [reply]
Hooray, that worked. Thanks!

**Comment by charitarthchugh** [comment]
Homebrew on Linux (Linuxbrew) also works well. see https://brew.sh
